service-templates/apigw-lambda-svc/v1/instance_infrastructure/outputs.tf

```diff
@@ -1,9 +1,11 @@
+# tflint-ignore: terraform_naming_convention - matching case of CloudFormation equivalent for testing purposes
```

Comment:
These are the ignore rules I mentioned - without them, these lines would show up in the `tflint` errors. Adding a bit of an explanation just helps our future-selves as well as users to know why we are deviating from the norm